includes/virtual-machines-disks-ultra-ssd.md

```diff
@@ -18,7 +18,7 @@ Azure Ultra SSD (preview) delivers high throughput, high IOPS, and consistent lo
 
 **Managed Disks**: Ultra SSDs are only available as Managed Disks. Ultra SSDs cannot be deployed as an Unmanaged Disk or Page Blob. While creating a Managed Disk, you specify the disk sku type as UltraSSD_LRS and indicate the size of disk, the IOPS, and throughput you need, and Azure creates and manages the disk for you.  
 
-**Virtual Machines**: Ultra SSDs are designed to work with all Premium SSD enabled Azure Virtual Machine SKUs, however at preview time the VM sizes will be limited to ES/DS v3 VM instances.
+**Virtual Machines**: Ultra SSDs are designed to work with all Premium SSD enabled Azure Virtual Machine SKUs, however as it currently in preview the VM sizes will be limited to the VMs are sized as ES/DS v3.
```

Comment:
If you would mind changing this to: "however, as it is currently in preview" that would be appreciated! Then I will sign off.

Reply:
hi @roygara you're right :)
So I changed it as you requested. Thank you very much !